checks/binary_artifact.go

```diff
@@ -24,10 +24,11 @@ import (
 // CheckBinaryArtifacts is the exported name for Binary-Artifacts check.
 const CheckBinaryArtifacts string = "Binary-Artifacts"
 
-//nolint
+//nolint:gochecknoinits
 func init() {
 	supportedRequestTypes := []checker.RequestType{
 		checker.CommitBased,
+		checker.FileBased,
```

Comment:
there's the remaining problem that this test would be enabled on pull_request in the Action, but should not because it needs API access? (Different problem, I suppose.. I recall there's an existing issue for it)

Reply:
Yes, that did come to mind. If run locally on PRs, then we would warn about something that might be ok after merging.
Since PRs are experimental, I'm not sure I want to block on this currently. I can track down the relevant issue and leave a comment.